packages/ckeditor5-utils/src/version.ts

```diff
@@ -15,6 +15,8 @@ const version = '37.1.0';
 
 export default version;
 
+export const releaseDate = new Date( 2023, 3, 19 );
```

Comment:
It is a good idea to leave reminder in inline comment that month is 0-based. I personally remember that one of `Date()` arguments is 0-based but I never remember which one 😂